src/VisualStudio/IntegrationTests/EditorTestFixture.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Roslyn.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities;
+using Roslyn.VisualStudio.Test.Utilities.Common;
+using Roslyn.VisualStudio.Test.Utilities.Input;
+using Roslyn.VisualStudio.Test.Utilities.OutOfProcess;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests
+{
+    public abstract class EditorTestFixture : IDisposable
```

Comment:
`Fixture` has specific meaning in regards to xUnit tests that this doesn't seem to satisfy (https://xunit.github.io/docs/shared-context.html#class-fixture). It might cause some confusion later on.